Added two translated file from  DP-GEN/learnDoc/*

Mile-Away opened the pull request:
Added the translated file from  DP-GEN/learnDoc/*, as the file structure as in English totorials for deepmodeling.

为 DeepModeling 社区 DP-GEN 的内容增加了两个中文翻译的文档。参考的是 DeepModeling 社区英文版的 DP-GEN 的文档。文件目录格式与英文 Tutorial 的目录结构一致，都放在 DP-GEN/learndoc/ 下。

DP-GEN_handson.md 中有一处粉底标注的文字，是我不太确定的地方。其它基本无疑问，请校对。

Mile-Away committed:
Added the translated file from  DP-GEN/learnDoc/*

Added the translated file from  DP-GEN/learnDoc/*, as the file structure as in English totorials for deepmodeling.

LiangWenshuo1118 commented:
@Q-Query 
line 5, 并发学习 -> 同步学习
line 31, 从头算ab initio弛豫 -> 从头算(ab initio)弛豫. Check the files and put the English word in brackets. 
line 31, AIMD仿真 -> AIMD模拟
line 242, DeepMD-kit -> DeePMD-kit. Please ensure that this error is not found elsewhere.
line 311, KSSPACE -> KSPACING
line 371, DeeOMD-kit -> DeePMD-kit
line 373, CP，MV -> cp，mv
line 496, DeePMD套件 -> DeePMD-kit

Mile-Away committed:
Correct the mistakes as comment in last PR

Thanks for your correction. I've corrected the mistakes as the comment in the PR.

For mistake 1 and 4, I've check the content for many times. Generally, the English word as the appendant of Chinese word has put into brackets. And all of the error, DeepMD-kit, has replaced with DeePMD-kit.

Mile-Away commented:
Thanks for your correction. I've corrected the mistakes as the comment in the PR.

For mistake 2 and 4, I've check the content for many times. The English word as the appendant of Chinese word has put into brackets. And all of the error, DeepMD-kit, has replaced with DeePMD-kit.
